README.md

```diff
@@ -103,8 +103,7 @@ Once the testing environment is fully operational, the following services will b
 
 ## Roadmap
 
-1. Add more components, such as LDAP, Kerberos, Flink, HBase, etc.
-2. This project is in very early stage, and the current way is not flexiable enough to support various testing scenarios.
-For example, there is no switch to turn on/off each components by configuration. To address such issues, we may want
-to leverage the template engine like Ansible plus Jinja2 to generate the docker-compose.yml file and configuration files
-for each component, so that user could easily customize the testing environment by modifying the configurations.
+1. Add more components, such as LDAP, Kerberos, HBase, etc.
+2. Fully templatized. Leverage Ansible and Jinja2 to templatize the Dockerfiles, shell scripts, and configuration files, so that user can easily customize the testing environment by modifying the configurations, e.g. only enable the a subset of components, change the version of the components.
```

Comment:
```suggestion
2. Fully templatized. Leverage Ansible and Jinja2 to templatize the Dockerfiles, shell scripts, and configuration files, so that users can easily customize the testing environment by modifying the configurations, e.g. only enabling a subset of components, and changing the version of the components.
```